subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/algorithm/hived_algorithm.go

```diff
@@ -888,8 +893,10 @@ func generatePodScheduleResult(
 	} else {
 		// we find the selected node after the preemption is done, otherwise the preemption victims
 		// may cause the selected node to be excluded from the suggested nodes
-		affinityGroupBindInfo, selectedNode, selectedGpuIndices, cellChain := generateAffinityGroupBindInfo(
+		affinityGroupBindInfo, selectedNode, selectedGpuIndices, cellChain, physicalPlacementString, virtualPlacementString := generateAffinityGroupBindInfo(
 			groupPhysicalPlacement, groupVirtualPlacement, cellLevelToType, currentGpuNum, currentPodIndex, group, groupName, suggestedNodeSet)
+		klog.Infof("Physical placement: %v", physicalPlacementString)
+		klog.Infof("Virtual placement: %v", virtualPlacementString)
```

Comment:
May need a common func to convert placements to exposable structure, then you can just toJson(it)